api/tests/conftest.py

```diff
@@ -410,6 +410,12 @@ def api_auth_token(monkeypatch, all_api_auth_tokens):
     return auth_token
 
 
+@pytest.fixture
+def api_user_key_token(user_api_key):
+    """Fixture providing a valid API user key for testing CG routes."""
+    return user_api_key.key_id
```

Comment:
There is an existing `user_api_key_id` fixture that is this already - it's just not quite with the other auth, not sure why it's at the top of the file.